app/debug_overlay.conf

```diff
@@ -1 +1,6 @@
 CONFIG_DEBUG=y
+CONFIG_ASSERT=y
+CONFIG_STACK_SENTINEL=y
+CONFIG_SYS_HEAP_VALIDATE=y
+CONFIG_SPIN_VALIDATE=y
```

Comment:
I think `SPIN_VALIDATE` is "y" by default when `ASSERT` is enabled, but setting it again shouldn't hurt